rust-bindings/fuzz/fuzz_targets/fuzz_filter.rs

```diff
@@ -0,0 +1,7 @@
+#![no_main]
+
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: Vec<&[u8]>| {
+    let _ = chiabip158::encode_filter(&data);
```

Comment:
shouldn't you assert some properties of the result here? Maybe you don't have any bindings for testing membership in a filter.

Reply:
I can add this as well once I setup proper testing and CI